pkg/runtime/runtime.go

```diff
@@ -695,6 +690,110 @@ func (a *DaprRuntime) beginPubSub(subscribeCtx context.Context, name string, ps
 		}
 	}
 
+	for _, actorConfigSubscriptions := range a.appConfig.Pubsub {
+		if actorConfigSubscriptions.PubSubName == name {
+			allowed := a.isPubSubOperationAllowed(name, actorConfigSubscriptions.Topic, a.scopedSubscriptions[name])
```

Comment:
I wonder if we can reuse the subscription code above. It is odd that we need to implement DLQ and TTL again here.